### PR TITLE
chore: improve logging for 6.1.x

### PR DESCRIPTION
**Pull request by pgaref**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java [hunk 1]
```diff
@@ -324,17 +325,17 @@ void fetchAndRunCommands() {
     }
   }
 
+  @SuppressFBWarnings("DMI_INVOKING_TOSTRING_ON_ARRAY")
   private void executeStatement(final QueuedCommand queuedCommand) {
-    final String commandStatement =
-        queuedCommand.getAndDeserializeCommand(commandDeserializer).getStatement();
-    LOG.info("Executing statement: " + commandStatement);
+    final String commandId = queuedCommand.getCommandId().toString();
```
Comment (vcrfxia) [inline ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java:330]: This method is marked as visible for testing: https://github.com/confluentinc/ksql/blob/9c74ff7cf9880317a55690ca291995b7315179c2/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/QueuedCommand.java#L59

Looks like you're meant to use `queuedCommand. getAndDeserializeCommandId()` instead, and then if you `toString()` you should be able to remove the `DMI_INVOKING_TOSTRING_ON_ARRAY` suppression as well.

**Review by vcrfxia — APPROVED** [review]
Thanks @pgaref -- LTGM besides the comment inline.

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java [hunk 2]
```diff
@@ -327,7 +327,7 @@ void fetchAndRunCommands() {
 
   @SuppressFBWarnings("DMI_INVOKING_TOSTRING_ON_ARRAY")
   private void executeStatement(final QueuedCommand queuedCommand) {
-    final String commandId = queuedCommand.getCommandId().toString();
+    final String commandId = queuedCommand.getAndDeserializeCommandId().toString();
```
Comment (vcrfxia) [inline ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java:330]: @pgaref can we remove the `DMI_INVOKING_TOSTRING_ON_ARRAY` FB suppression now?

Comment (pgaref) [reply ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java:330]: Thats why I commited this in the first place, and cherry-picking from 7.2 tricked me! Thanks :D

**Pull request merged**